app/styles/front.scss

```diff
@@ -616,22 +616,17 @@ input:checked + .btn, input:checked + .btn:hover, input:checked + .btn:active:ho
 .button-grid {
   display: flex;
   flex-wrap: wrap;
+  margin-left: -2px;
+  margin-right: -2px;
 
   label {
-    width: 33.33%;
+    width: calc(100% * 1/3 - 4px);
```

Comment:
> Partial support in Android Browser 4.4 refers to the browser lacking the ability to multiply and divide values.
([source](http://caniuse.com/#search=calc))
We can get 1% more support by using `33% - 4px` rather than `100% * 1/3`.
I think this `4px` should rather be something like `.5em` to make the spaces relative to the font size.